components/MobileNavigation.js

```diff
@@ -20,23 +20,53 @@ export default class MobileNavigation extends Component {
 
   state = {
     showMobileNav: false,
-    expandMobileNav: false
+    expandMobileNav: false,
+    nowSelectMenu: null,
+    scrollPosition: null
   }
 
   componentDidMount () {
+    const { navs } = this.props
     window.addEventListener('scroll', this.handleScroll)
+    const scrollPosition = navs.reduce((obj, nav, index) => {
+      const navScroll = {
+        name: nav.label,
+        scroll: document.getElementById(nav.label.toLowerCase()).getBoundingClientRect().top + window.scrollY
+      }
+      obj[index] = navScroll
+      return obj
+    }, [])
+    this.setState({
+      scrollPosition: scrollPosition
+    })
   }
 
   componentWillUnmount () {
     window.removeEventListener('scroll', this.handleScroll)
   }
 
   handleScroll = (e) => {
+    const _docHeight = (document.height !== undefined) ? document.height : document.body.offsetHeight
+    let result
```

Comment:
Hi am almost complete all thing your review but i have one question.Do you thing about switch result variable from let to const ? I think Because result variable want able to change value.If i declare const in condition scope.i will can't call result const outside condition scope to set new state.Any idea. Thankyou.